assets/svelte/components/PageAstNode.svelte

```diff
@@ -97,6 +125,41 @@
       },
     }
   }
+
+  function bindIfSelected(el: HTMLElement, isSelected: boolean) {
+    if (isSelected) {
+      $selectedDomElement = el;
+      initSelectedElementMenuPosition(el);
+    }
+
+    return {
+      update(isSelected) {
+        if (isSelected) {
+          $selectedDomElement = el;
+          initSelectedElementMenuPosition(el);
+        }
+      },
+      destroy() {
+        if (isSelected) {
+          $selectedDomElement = null;
+        }
+      }
+    }
+  }
+
+  let selectedElementStyle = '';
+  $: {
+    if (isSelectedNode && $selectedElementMenu && $selectedElementMenu.mouseMovement) {
+      let {x, y} = mouseDiff($selectedElementMenu.mouseMovement)
+      if ($selectedElementMenu.dragDirection === 'vertical') {
```

Comment:
I played around a little bit, and it sometimes feels a bit confusing what direction it will move and why. Can you explain, or should we make it more obvious to the user?